data/map planets.txt

```diff
@@ -2980,8 +2980,6 @@ planet "New China"
 	spaceport `	There are always locals hanging around the port, some of them mere children, who are trying to find a cheap ride off-planet.`
 	outfitter "Common Outfits"
 	outfitter "Ammo South"
-	outfitter "Delta V Basics"
-	outfitter "Kraz Basics"
```

Comment:
Do we have any other outfitters that appear outside of their usual region? When it comes to the basics outfitters, I don't see an issue with them being sold in more distant locations. At the very least, it'd make more sense to replace these with something else than make New China a worse location to go to.

Reply:
The one that comes to mind are pirate worlds which stock stuff from everywhere, but nothing else when it comes to outfitters.

I'll repeat what I suggested in the description, perhaps I could make a unique outfitter or something that shows up in the immediate area, at least something that explains why New China has these outfitters.

Reply:
I don't see much use in adding a unique outfitter, nor do I see much of a need to have this be explained.

Reply:
So is there an issue with this still?

Reply:
These should still be replaced instead of entirely removed if any change is to occur. Other nearby planets use syndicate and lovelace basics.